pkgs/native_assets_cli/test/api/resource_data.dart

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:native_assets_cli/src/model/resource_identifiers.dart';
```

Comment:
Should this be in tests/model/ instead?

(Or should this file be removed from this PR completely?)

Reply:
I don't think it should be removed, just hidden.